types/assertions.d.ts

```diff
@@ -124,7 +124,7 @@ export interface NightwatchCommonAssertions<ReturnType> {
 	 *
 	 */
 	elementsCount(
-		selector: Definition,
+		selector: ScopedSelector,
```

Comment:
Shouldn't this be `Definition` as assert commands could also take in `WebElement` instance?

Reply:
Passing a `WebElement` works with all commands except `elementPresent` and `elementsCount`, and it also does not make sense to pass a `WebElement` to these commands, because having a `WebElement` to pass automatically tells that the element is present (otherwise we wouldn't have a `WebElement` instance for that element) and the count is 1.